### PR TITLE
Creating a signing target that will handle resolver files.

### PR DESCRIPTION
**Pull request by livarcocc**
@dotnet/dotnet-cli @nguerrera @srivatsn 

@MattGertz This is a infra-structure only change and require for the resolver work. I will merge when it turns green.

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build
@dotnet-bot Test Ubuntu x64 Release Build
@dotnet-bot Test Ubuntu16.04 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

File: build/Signing.proj [hunk 1]
```diff
@@ -18,6 +18,7 @@
       <OutputDirectory Condition="'$(OutputDirectory)' == ''">$(BaseOutputDirectory)/stage2</OutputDirectory>
       <CompilationDirectory Condition="'$(CompilationDirectory)' == ''">$(BaseOutputDirectory)/stage2compilation</CompilationDirectory>
       <PackagesDirectory Condition="'$(PackagesDirectory)' == ''">$(BaseOutputDirectory)/packages</PackagesDirectory>
+      <SdkResolverOutputDirectory>$(BaseOutputDirectory)/intermediate/MSBuildSdkResolver</SdkResolverOutputDirectory>
```
Comment (nguerrera) [inline build/Signing.proj]: Why is this duplicated from OutputDirectories.props?

**Comment by nguerrera** [comment]
👍 Thanks for fixing this.

**Pull request merged**